src/quo2/components/settings/data_item/style.cljs

```diff
@@ -0,0 +1,73 @@
+(ns quo2.components.settings.data-item.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn container
+  [size card? theme blur?]
+  {:width              335
+   :flex-direction     :row
+   :justify-content    :space-between
+   :padding-vertical   (if (= size :default)
```

Comment:
Just a nitpick, in my taste it is better to be a one-liner for readability)

Reply:
Done